static/src/javascripts-legacy/projects/common/modules/email/email-article.js

```diff
@@ -205,16 +210,41 @@ define([
 
                 storage.session.set('email-sign-up-seen', 'true');
             }
-        };
+        }
 
     return {
         init: function () {
             if (emailRunChecks.allEmailCanRun()) {
                 // First we need to check the user's email subscriptions
                 // so we don't insert the sign-up if they've already subscribed
                 emailRunChecks.getUserEmailSubscriptions().then(function () {
-                    // Get the first list that is allowed on this page
-                    addListToPage(find(listConfigs, emailRunChecks.listCanRun));
+                    if (ab.isParticipating({id: 'TailorRecommendedEmail'}) &&
```

Comment:
The init function is getting quite long.  It might be clearer as:
```
if (ab.isParticipating({id: 'TailorRecommendedEmail'}) {
  switch (ab.getTestVariantId) {
    case 'tailor-recommended': tailorInTest(); break;
    case 'control': tailorControl(); break;
    case default: addListToPage(find(listConfigs, emailRunChecks.listCanRun)); break;
  }
} else {
  addListToPage(find(listConfigs, emailRunChecks.listCanRun));
}
```

But feel free to disagree!